fuge/system.yml

```diff
@@ -52,6 +52,18 @@ nodezoo_travis:
   net: host
   restart: always
 
+nodezoo_updater:
```

Comment:
I did not submit a restart setting for updater, or dequeue. @AdrianRossouw what should this be?